tests/python/ColorSpaceTest.py

```diff
@@ -754,4 +790,112 @@ def check_processor_inv(self, p):
         # Test linear color space to_ref direction.
         cfg.setInactiveColorSpaces("Linear ITU-R BT.709, Texture -- sRGB")
         p = OCIO.Config.GetProcessorFromBuiltinColorSpace(builtin_csname, cfg, src_csname)
-        check_processor_inv(self, p)
\ No newline at end of file
+        check_processor_inv(self, p)
+
+
+        editableCfg = copy.deepcopy(cfg)
+        builtinConfig = OCIO.Config.CreateFromFile("ocio://default")
+
+        #
+        # Test IdentifyBuiltinColorSpace.
+        #
+
+        editableCfg.setInactiveColorSpaces("")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "ACEScg")
+        self.assertEqual(csname, "ACES cg")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "sRGB - Texture")
+        self.assertEqual(csname, "Texture -- sRGB")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "ACES2065-1")
+        self.assertEqual(csname, "ref_cs")
+
+        editableCfg.setInactiveColorSpaces("Texture -- sRGB, ref_cs")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "Linear Rec.709 (sRGB)")
+        self.assertEqual(csname, "Linear ITU-R BT.709")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "ACEScct")
+        self.assertEqual(csname, "CS Transform color space")
+
+        csname = cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "lin_ap1")
+        self.assertEqual(csname, "ACES cg")
+
+        # Display-referred spaces are not supported unless the display-referred interchange
+        # role is present.
+
+        with self.assertRaises(OCIO.Exception) as cm:
+            cfg.IdentifyBuiltinColorSpace(editableCfg, builtinConfig, "sRGB - Display")
+        self.assertEqual(
```

Comment:
Should this be indented under the `with` (same apply below)?

Reply:
The current code is correct. The goal of the `with` statement is to capture the exception, set the `cm ` variable to the exception message. Once the exception is thrown, then do a check on that message. If we were to put that under the `with` statement, it would not be executed because `cfg.IdentifyBuiltinColorSpace` would throw first.

Reply:
You are perfectly right, sorry about this confusion!